libmscore/style.h

```diff
@@ -968,6 +968,8 @@ enum class Sid {
       glissandoFrameBgColor,
       glissandoLineWidth,
       glissandoText,
+      glissandoEaseIn,
+      glissandoEaseOut,
```

Comment:
You added 2 here, but not in style.cpp, this causes the crash resp. assertion failure

Reply:
Delete those 2, you don't use them anywhere

Reply:
I see. I added those two styles in the beginning of my exploration but they caused the "apply as style" button to appear in the inspector so I removed it. But obviously, I did not remove it everywhere. Thanks for pointing to it. I'll fix that.